tools/edrumulus_gui.py

```diff
@@ -48,7 +48,8 @@
 
 # tables
 pad_types    = ["PD120", "PD80R",  "PD8", "FD8", "VH12", "VH12CTRL", "KD7", "TP80",     "CY6",    "CY8", "DIABOLO12", \
-                "CY5",   "HD1TOM", "PD6", "KD8", "PDX8", "KD120",    "PD5", "PDA120LS", "PDX100", "KT10"]
+                "CY5",   "HD1TOM", "PD6", "KD8", "PDX8", "KD120",    "PD5", "PDA120LS", "PDX100", "KT10", "MPS750X_TOM", \
```

Comment:
Today, I did mayor changes to the pad type setting in the GUI. Previously, the sorting of the pad types were a mess. Now it is possible to sort them meaningfully like mesh pads first, then rubber pads, then cymbals, etc. As a consequence, your merge request now has conflicts to solve at this place.